### PR TITLE
Adding checkbox functionality

### PR DESCRIPTION
**Pull request by lawrencekgrant**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "attainia-web-components",
-  "version": "2.2.12",
+  "version": "2.3.12",
```
Comment (mtomcal) [inline package.json]: Patch version will reset on a new minor version to 2.3.0.

Comment (lawrencekgrant) [reply package.json]: Changed to 2.3.0.

**Comment by mtomcal** [comment]
Finishing up some travis info. Will do a proper looksee in a couple minutes. Thanks!

**Comment by mtomcal** [comment]
Sent you some slack on UX stuff for white checkmarks. The checkbox port from carbon looks great overall in the tests and implementation.

**Pull request merged**